tests/unit/anchore_engine/services/policy_engine/engine/feeds/test_config.py

```diff
@@ -124,8 +124,8 @@ def test_get_selected_configs_to_sync_defaults(provider, test_config, expected):
         pytest.param(
             LegacyProvider,
             {"provider": "legacy", "sync": {"data": {"vulndb": {"enabled": True}}}},
-            {"vulndb"},
-            id="valid-legacy-vulndb",
+            set(),
```

Comment:
why is this invalid now? not blocking comment, just don't understand why this would change.

Reply:
Because I've removed it from the default config here https://github.com/anchore/anchore-engine/pull/1072/files#diff-407ba06f155493ba97d77fe4c33a2a7cd64efac42dfb1f25d03ec9bb3e01fb60L133

It'll be added to config in the downstream provider that actually support the feed